argmin/src/solver/quasinewton/lbfgs.rs

```diff
@@ -402,6 +402,7 @@ where
         };
 
         // L-BFGS two-loop recursion
+        #[allow(clippy::redundant_clone)]
```

Comment:
This is not really a redundant clone, because line 428 needs the unmodified previous gradient.

Reply:
Interesting that clippy doesn't catch that.